frontend/src/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionRegisteredDeploymentsView.tsx

```diff
@@ -22,6 +23,12 @@ const ModelVersionRegisteredDeploymentsView: React.FC<
     return (
       <EmptyModelRegistryState
         title="No deployments from model registry"
+        headerIcon={() => (
+          <img
+            src={typedEmptyImage(ProjectObjectType.registeredModels, 'MissingDeployment')}
+            alt="missing deployment"
+          />
+        )}
```

Comment:
I don't find a testid here, so I think we don't have a Cypress test for empty-registered-deployments. Can you please add the test for this?

Reply:
there's a test-id a couple lines below this, and the test checking for it is in `modelVersionDetails.cy.ts:192`